components/datetime/tests/fixtures/tests/components_with_zones.json

```diff
@@ -138,31 +88,5 @@
                 "en": "Tuesday, January 21, 2020, 08:25:07 GMT+05:00"
             }
         }
-    },
-    {
-        "description": "Date time example that includes a week-of for a locale with plural variants as well as one with non default WeekData",
```

Comment:
Even though week-of-year is dead right now, it should be added back in a future release. #5643